tests/utils/jenkins.go

```diff
@@ -130,6 +130,22 @@ func RetrieveJenkinsBuildStagesForBuild(jenkinsNamespace string, buildName strin
 		count++
 	}
 
+	// in case the the build was sort of never really started - get the jenkins pod log, maybe there
+	// is a plugin / sync problem?
+	if build.Status.Phase == v1.BuildPhaseNew || build.Status.Phase == v1.BuildPhasePending {
+		// get the jenkins pod log
+		stdoutJPod, stderrJPod, errJPod := RunScriptFromBaseDir(
+			"tests/scripts/print-jenkins-pod-log.sh",
+			[]string{
+				jenkinsNamespace,
+			}, []string{})
+		if errJPod != nil {
+			fmt.Printf("Error creating getting jenkins pod logs: %s", errJPod)
```

Comment:
might want to print `stderrJPod` as well as that might contain the reason why it failed?